src/molecules/LiistPlaceCardNew.js

```diff
@@ -210,31 +210,55 @@ export class LiistPlaceCardNew extends LitElement {
     super();
   }
 
+  // after the component is first rendered in its initial state (displayed to user)
+  // this function gets executed
+  firstUpdated() {
+    this.setImage();
```

Comment:
firstUpdated() is a nice function build in lit-element, after the coponent has rendered this function is executed.